scripts/setup_providers.sh

```diff
@@ -2,109 +2,38 @@
 __dir=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 . ${__dir}/vars/variables.sh
 LOGS_DIR=${__dir}/../testutil/debugging/logs
-mkdir -p $LOGS_DIR
```

Comment:
I think you didnt want to commit these changes.

Reply:
That's why this commit is called WIP :)

Reply:
It was my saved environment

Reply:
please dont commit to github local environment changes. :)